src/plugins/visualizations/server/migrations/visualization_common_migrations.ts

```diff
@@ -41,6 +41,19 @@ export const commonAddDropLastBucketIntoTSVBModel = (visState: any) => {
   return visState;
 };
 
+export const commonAddDropLastBucketIntoTSVBModel714Above = (visState: any) => {
+  if (visState && visState.type === 'metrics') {
+    return {
+      ...visState,
+      params: {
+        ...visState.params,
+        drop_last_bucket: visState.params.drop_last_bucket ?? 1,
+      },
```

Comment:
nit: I think that if you invert the result is the same:

```suggestion
      params: {
        drop_last_bucket: 1,
        ...visState.params
      },
```